test/e2e/webhook_e2e_test.go

```diff
@@ -415,7 +415,7 @@ var _ = Describe("CSVs with a Webhook", func() {
 
 			// Make sure old resources are cleaned up.
 			Eventually(func() bool {
-				return csvExists(crc, csv.Spec.Replaces)
+				return csvExists(testNamespace, crc, csv.Spec.Replaces)
```

Comment:
#2718 landed since this PR has opened - do we need to update this to use the test generated namespace?

Reply:
(I fixed this in the latest force push)